---
title: "Comprehensive structural and substrate specificity classification of the Saccharomyces cerevisiae methyltransferome."
date: 2011-08-01
publishDate: 2022-06-11T17:26:08.304963Z
authors: ["Tomasz Wlodarski", "Jan Kutner", "Joanna Towpik", "Lukasz Knizewski", "Leszek Rychlewski", "Andrzej Kudlicki", "Maga Rowicka", "Andrzej Dziembowski", "Krzysztof Ginalski"]
publication_types: ["2"]
abstract: "Methylation is one of the most common chemical modifications of biologically active molecules and it occurs in all life forms. Its functional role is very diverse and involves many essential cellular processes, such as signal transduction, transcriptional control, biosynthesis, and metabolism. Here, we provide further insight into the enzymatic methylation in S. cerevisiae by conducting a comprehensive structural and functional survey of all the methyltransferases encoded in its genome. Using distant homology detection and fold recognition, we found that the S. cerevisiae methyltransferome comprises 86 MTases (53 well-known and 33 putative with unknown substrate specificity). Structural classification of their catalytic domains shows that these enzymes may adopt nine different folds, the most common being the Rossmann-like. We also analyzed the domain architecture of these proteins and identified several new domain contexts. Interestingly, we found that the majority of MTase genes are periodically expressed during yeast metabolic cycle. This finding, together with calculated isoelectric point, fold assignment and cellular localization, was used to develop a novel approach for predicting substrate specificity. Using this approach, we predicted the general substrates for 24 of 33 putative MTases and confirmed these predictions experimentally in both cases tested. Finally, we show that, in S. cerevisiae, methylation is carried out by 34 RNA MTases, 32 protein MTases, eight small molecule MTases, three lipid MTases, and nine MTases with still unknown substrate specificity."
featured: false
publication: "*Plos One*"
url_pdf: "http://dx.doi.org/10.1371/journal.pone.0023168"
doi: "10.1371/journal.pone.0023168"
---

